src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,68 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String OPERATION_SUPPLY = "supply";
+    private static final String OPERATION_BUY = "buy";
+    private static final String OPERATION_RESULT = "result";
+    private static final String SPLIT_REGEX = ",";
+    private static final int INDEX_FOR_OPERATION_TYPE = 0;
+    private static final int INDEX_FOR_AMMOUNT = 1;
```

Comment:
```suggestion
    private static final int AMOUNT_INDEX = 1;
```
usually it's the common naming (of course it's not critical for code, just a heads up🙂)

Reply:
thanks